index.bs

```diff
@@ -3514,7 +3514,7 @@ Maturity Stages on the Recommendation Track</h4>
 		<dd>
 			A Candidate Recommendation is a document that satisfies the technical
 			requirements of the Working Group that produced it and their dependencies,
-			and has already received wide review.
+			and has already received a wide review.
```

Comment:
I would not take this change. We use "wide review" as a set phrase, and it is uncountable. This is not a question of how many reviews we get, just of the fact that the activity of reviewing happens.